.travis.yml

```diff
@@ -43,6 +43,6 @@ script:
   - $ISLANDORA_DIR/tests/scripts/line_endings.sh sites/all/modules/islandora_solution_pack_newspaper
   - drush coder-review --reviews=production,security,style,i18n,potx,sniffer islandora_newspaper
   - phpcpd --names *.module,*.inc,*.test sites/all/modules/islandora_solution_pack_newspaper
-  - drush test-run --uri=http://localhost:8081 "Islandora Newspaper"
+  - php scripts/run-tests.sh --php `phpenv which php` --url http://localhost:8081 --verbose --class IslandoraNewspaperPageIngestTestCase
```

Comment:
@jonathangreen i changed this to the fixed version. Thanks! Went for 4.0 (currently master, my bad, i should have directly targeted 4.0 from the beginnig) because the OCR i was getting in the islandora VM 1.8 (using 3.02 and old trained data) was not passing the assertion tests. Some of the "TIFF" written in the test.tiff file where detected as T|FF (with a pipe).